op-guide/tikv-config-file.md

```diff
@@ -0,0 +1,760 @@
+- The duration to log a "missing" message and report to the monitoring system if the peer is considered "missing"
+- Default: "10m"
+
+### `peer-stale-state-check-interval`
+
+- The interval to check a peer's state
+- Default: "5m"
+
+### `snap-apply-batch-size`
+
+- The maximum size to apply keys in the snapshot file in batch
+- Default: "10MB"
+
+### `consistency-check-interval`
+
+- The interval to check whether all the peers are consistent or not
+- Default: "0s"
+
+### `report-region-flow-interval`
+
+- The interval to report the region's read/write flow to the PD
+- Default: "1m"
+
+### `raft-store-max-leader-lease`
+
+- The max time for a Raft leader to provice local read (not through Raft) in the lease
+- Default: "9s"
+
+### `right-derive-when-split`
+
+- The flag to let the new right/or left split region inherit the meta data from the parent
+- Default: true
+- The region will be split into two new regions left + right. If true, the right will inherit from the parent
+
+### `allow-remove-leader`
+
+- The flag to allow removing the Raft leader in the RemoveNode request 
+- Default: false
+
+### `merge-max-log-gap`
+
+- The maximum gap to allow merge or not
+- Default: 10
+- If two peers's log gap is larger than the threshold, the `merge` request will be skipped
+
+### `merge-check-tick-interval`
+
+- The interval to check whether a region needs to be merged with other region.
+- Default: "10s"
+
+### `use-delete-range`
+
+- Use RocksDB `DeleteRange` to clear the data or not
+- Default: false
+- It is not suggested to enable it now
+- After performance problem of `DeleteRange` is fixed, the flag will be enabled by default
+
+### `cleanup-import-sst-interval`
+
+- The interval to cleanup the stable SST files generated in the load phase
+- Default: "10m"
+
+## `coprocessor`
+### `split-region-on-table`
+
+- To split the region by table or not
+- Default: true
+- If enabled, the region can only hold one table's data. If a region has multipy tables' data, the region will be split
+
+### `region-max-size`
+
+- The maximum size for a region
+- Default: "144MB"
+- If a region's size exceeds this, the region will be split
+
+### `region-split-size`
+
+- The size for a region to be split
+- Default: "96MB"
+- If a region's size exceeds `region-max-size`, the region will be split, a split key will be chosen after scanning `region-split-size` data
+
+## `rocksdb`
+
+### `wal-recovery-mode`
+
+- WAL recovery mode
+- Default: 2
+- 0 : TolerateCorruptedTailRecords, tolerate incomplete record in trailing data on all logs
+- 1 : AbsoluteConsistency, We don't expect to find any corruption in the WAL
+- 2 : PointInTimeRecovery, Recover to point-in-time consistency
+- 3 : SkipAnyCorruptedRecords, Recovery after a disaster
+
+### `wal-dir`
+
+- The directory for the RocksDB WAL
+- Default: ""
+
+### `wal-ttl-seconds`
+
+- The time to live seconds for a WAL
+- Default: 0
+- Every TTL seconds / 2, the WAL which is older than `wal-ttl-seconds` will be deleted
```

Comment:
It would be clearer to change this sentence to: 

WAL files will be checked every `wal-ttl-seconds`/2 and those that are older than `wal-ttl-seconds` will be deleted.